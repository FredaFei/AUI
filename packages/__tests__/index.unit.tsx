import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../button'

describe('Button', () => {
  it('first test', () => {
    const json = renderer.create(<Button />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
