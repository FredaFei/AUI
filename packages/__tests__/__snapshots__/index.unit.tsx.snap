// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button first test 1`] = `
<div>
  我是按钮
</div>
`;
